tools/config/Tomb1Main_ConfigTool/Resources/Lang/es.json

```diff
@@ -0,0 +1,322 @@
+    "label_about_details": "Visite la página de Tomb1Main GitHub para obtener más información.",
+    "label_no_file": "Ningún archivo seleccionado",
+    "label_saved": "Guardado",
+    "label_unsaved": "No guardado",
+    "link_github": "Ir a GitHub",
+    "menu_file": "_Archivo",
+    "menu_help": "_Ayuda",
+    "menu_tools": "_Herramientas",
+    "msgbox_unsaved_changes": "¿Quieres guardar los cambios que has hecho?",
+    "msgbox_unsaved_changes_reload": "¿Seguro que quieres volver a cargar y perder los cambios que has hecho?",
+    "spinner_decrease": "Disminuir valor",
+    "spinner_increase": "Aumentar valor",
+    "spinner_msg_comparison_failed": "El valor de entrada debe estar entre {0} y {1}",
+    "spinner_msg_invalid_number": "La cadena de entrada no es un número válido",
+    "window_title_about": "Acerca de la herramienta de configuración de Tomb1Main",
+    "window_title_main": "Herramienta de configuración Tomb1Main"
+  },
+  "Enums": {
+    "airbar_mode": {
+      "default": "Por defecto",
+      "flashing-only": "Solo intermitente",
+      "never": "Nunca",
+      "ps1": "PS1"
+    },
+    "healthbar_mode": {
+      "always": "Siempre",
+      "default": "Por defecto",
+      "flashing-only": "Solo intermitente",
+      "flashing-or-default": "Intermitente o por defecto",
+      "never": "Nunca",
+      "ps1": "PS1"
+    },
+    "menu_style": {
+      "pc": "PC",
+      "ps1": "PS1"
+    },
+    "screenshot_format": {
+      "jpg": "JPEG",
+      "png": "PNG"
+    },
+    "ui_color": {
+      "blue": "Azul (predeterminado para aire)",
+      "blue2": "Azul (alternativo)",
+      "gold": "Oro (predeterminado para salud)",
+      "gold2": "Oro (alternativo)",
+      "green": "Verde",
+      "grey": "Gris",
+      "pink": "Rosa",
+      "red": "Rojo",
+      "silver": "Plata"
+    },
+    "ui_location": {
+      "bottom-center": "Inferior central",
+      "bottom-left": "Inferior izquierda",
+      "bottom-right": "Inferior derecha",
+      "top-center": "Superior central",
+      "top-left": "Superior izquierda",
+      "top-right": "Superior derecha"
+    }
+  },
+  "Properties": {
+    "airbar_color": {
+      "Description": "Color de la barra de aire.",
+      "Title": "Color de la barra de aire"
+    },
+    "airbar_location": {
+      "Description": "Ubicación donde se muestra la barra de aire.",
+      "Title": "Ubicación de la barra de aire"
+    },
+    "airbar_showing_mode": {
+      "Description": "Cambia la forma en que se muestra la barra de aire.\n\n- Predeterminado: muestra la barra de aire solo en el agua y parpadea la barra de aire cuando el oxígeno de Lara está al 20% o menos (TombATI).\n\n- Solo intermitente: muestra la barra de aire *solo* cuando el oxígeno de Lara está al 20% o menos (para carreras de desafío).\n\n- Nunca: nunca mostrar la barra de aire (para carreras de desafío).\n\n- PS1: muestra la barra de aire en circunstancias predeterminadas sin parpadeo.",
```

Comment:
Would it be possible to remove the extra line breaks? I think it would appear tidier and remain consistent with English and French.
![air](https://user-images.githubusercontent.com/33758420/223534751-313e231f-1335-4a1c-805c-273c29923bf1.png)
![air_en](https://user-images.githubusercontent.com/33758420/223534779-36691679-a540-4218-acfc-31af304d05c1.png)